pyo3-derive-backend/src/method.rs

```diff
@@ -47,10 +47,10 @@ pub fn get_return_info(output: &syn::ReturnType) -> syn::Type {
 
 impl<'a> FnSpec<'a> {
     /// Parser function signature and function attributes
-    pub fn parse(
+    pub fn parse<'b>(
```

Comment:
Is this required or can this be an implicit lifetime?

Reply:
it can probably be implicit, it's just required to not be `'a`.

Reply:
fixed